src/tokenizer.ts

```diff
@@ -2,7 +2,7 @@
  * Tokenizer
```

Comment:
For changes to the tokenizer, it would be good to see tests in `test/tokenizer/index.spec.ts` that capture that exact test case, e.g.:

```
  it('scan quoted identifier for mssql', function () {
    const actual = scanToken(initState('[aaa"bbb]'), 'mssql');
    const expected = {
      type: 'identifier',
      value: '[aaa"bbb]',
      start: 0,
      end: 13,
    };

    expect(actual).to.eql(expected);
  });
```